internal/resources/metal/connection/speed.go

```diff
@@ -2,32 +2,16 @@ package connection
 
 import (
 	"fmt"
-	"regexp"
 	"strconv"
 )
 
 var (
-	mega        uint64 = 1000 * 1000
-	giga        uint64 = 1000 * mega
-	SpeedFormat        = regexp.MustCompile(`^(\d+)((M|G)bps)$`)
+	mega int64 = 1000 * 1000
+	giga int64 = 1000 * mega
 )
 
-func speedStrToUint(speed string) (uint64, error) {
```

Comment:
It looks like we don't need this string to int conversion for speed any more because equinix-sdk-go takes a speed as a string on input.  Just to be safe we should probably keep `SpeedFormat` around for now and update the `speed` attribute in the resource schema to validate that the value matches that regexp.

Reply:
done